app/assets/stylesheets/static_pages.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```

Comment:
delete this file
Note: Raisl will auto generator scss, coffee and helper file. So, if you don't it, you should delete it.